skbio/stats/distance/_base.py

```diff
@@ -487,6 +487,34 @@ def to_data_frame(self):
         """
         return pd.DataFrame(data=self.data, index=self.ids, columns=self.ids)
 
+    @experimental(as_of="0.5.0-dev")
+    def to_series(self):
```

Comment:
Now that `to_series` returns the condensed form distances (i.e. only one triangle, excluding the diagonal), this method should be moved to `DistanceMatrix` since a scikit-bio distance matrix is guaranteed to be symmetric and hollow. `DissimilarityMatrix` can be non-hollow and/or asymmetric so condensed form isn't applicable here.